contrib/pyln-spec/bolt2/pyln/spec/bolt2/text.py

```diff
@@ -95,7 +95,7 @@
 - The `nLocktime` for the transaction has already been communicated, or is
   established via convention.
 - The transaction version is 2.
-- The each input's sequence number will be set to 0xFEFFFFFF (little endian), which signals RBF.
+- The each input's sequence number will be set to 0xFDFFFFFF (little endian), which signals RBF.
```

Comment:
This is not true, right?  And weird... 0xFFFFFFFE is the bitcoind default, and anyway this is now set as per tx_add_input?

And endian probably shouldn't be mentioned here anyway: you're describing the value, not the representation.

Reply:
This should have gone away when I added the sequence to the spec, will update.